index.html

```diff
@@ -49,10 +49,86 @@ <h2>LET'S CONNECT</h2>
               <a href="https://github.com/isaac-ej"><img src="img/Vector (1).svg" alt=""></a>
             </li>
           </ul>
-        
         </section>
-   </header>
+     </header>
+       <main class="card-container">
+         <!-- Work section -->
+         <section class="container">
+          <div class="bg">
+            <div class="snap">
+              <img src="/img/Snapshoot Portfolio.png">
+            </div>
+            <div class="left-block">
+              <div class="primary-text">
+                <h5 class="tonic">TONIC </h5>
+                <div class="frame">
+                  <p>Canopy</p>
+                  <ul>
+                    <li><div class="spans"></div></li>
+                    <li>Backend Dev</li>
+                    <li><p class="spans"></p></li>
+                    <li>2015</li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+            <div class="techs">
+              <ul>
+                <li>HTML</li>
+                <li> CSS</li>
+                <li>Javascript</li>
+              </ul>
+            </div>
+            <button>See Projects</button>
+          </div>
```

Comment:
- [ ] Great job on implementing the button! However, in order to align with the design requirements, please make the following updates:
- Apply the necessary CSS classes or styles to the button element to achieve the active, hover, and pressed button styles as specified in the design.

- [ ] Here is the required button style for each event
![image](https://github.com/Isaac-ej/Portfolio_Website/assets/68823331/4037e38d-b98c-4ad9-b54d-2b40d422d207)